module/canvas/detection-modes/detect-magic.mjs

```diff
@@ -0,0 +1,106 @@
+/**
+ * The detection mode for Detect Magic.
+ */
+export class DetectionModeDetectMagic extends DetectionMode {
```

Comment:
'Detect Magic' leaves a faint aura on a magical object/creature if one uses their action, which requires that the target is visible, but the Detection Mode is the other way around, only creating an effect for invisible targets. Could this be inverted or rather include both if possible?